### PR TITLE
Updates to Abl:imatinib example

### PR DESCRIPTION
**Pull request by jchodera**

### CODE REVIEW

File: examples/binding/abl-imatinib/explicit.yaml [hunk 1]
```diff
@@ -3,15 +3,15 @@ options:
   minimize: yes
   verbose: yes
   output_dir: .
-  number_of_iterations: 1000
+  number_of_iterations: 500
```
Comment (andrrizzi) [inline examples/binding/abl-imatinib/explicit.yaml:6]: Do we want to push this, or was this just a local change that got committed?

File: examples/binding/abl-imatinib/implicit.yaml [hunk 2]
```diff
@@ -3,8 +3,8 @@ options:
   minimize: yes
   verbose: yes
   output_dir: .
-  number_of_iterations: 1000
-  nsteps_per_iteration: 500
+  number_of_iterations: 250
+  nsteps_per_iteration: 2500
```
Comment (andrrizzi) [inline examples/binding/abl-imatinib/implicit.yaml:7]: Same here.

**Review by andrrizzi — APPROVED** [review]
Thanks! Looks good to me if those changes to `number_of_iterations` and `nsteps_per_iteration` are done on purpose.

File: examples/binding/abl-imatinib/explicit.yaml (hunk 1 above)
Comment (jchodera) [reply examples/binding/abl-imatinib/explicit.yaml:6]: These are local changes, but they shouldn't be a hinderance---they just try to balance simulation and non-simulation time for optimal performance.

Comment (andrrizzi) [reply examples/binding/abl-imatinib/explicit.yaml:6]: Great! I'll merge then.

**Pull request merged**